Assets/Examples/Test/Editor/Driver/Testforscene1TestSample.cs

```diff
@@ -278,50 +265,29 @@ public void TestGetComponentPropertyInvalidDeserialization()
             }
         }
 
-        [Test]
-        public void TestGetComponentPropertyNotFoundWithAssembly()
+        [TestCase("UnityEngine.UI.Text", "InvalidProperty", "UnityEngine.UI", "Property InvalidProperty not found")]
+        [TestCase("UNEXISTING", "m_Text", "UnityEngine.UI", "Component not found")]
+        [TestCase("UnityEngine.UI.Text", "m_Text", "UNEXISTING", "Assembly not found")] // -> this is commented because tere is not Assembly no found exception
```

Comment:
We have Assembly not found exceptio, also I don't know if this test is commented